src/inverseRobot.js

```diff
@@ -24,7 +24,24 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const keys = Object.keys(robot);
+  const values = Object.values(robot);
+  const res = {};
+  const valuesSet = new Set(values);
+
+  if (values.length !== valuesSet.size) {
```

Comment:
you can check on the cycle by keys: res has property key (if has return null). But your variant isn't bad 👌